templates/blocks/share_buttons.twig

```diff
@@ -37,6 +37,15 @@
             <span class="visually-hidden">{{ __( 'Share via', 'planet4-master-theme' ) }} Email</span>
         </a>
     {% endif %}
+    <!-- Bluesky -->
+    {% if share_platforms.bluesky %}
+        <a href="https://bsky.app/intent/compose?text={{ social.title|url_encode }}{% if share_text or social.description %} - {{ (share_text ?? social.description)|striptags|url_encode }}{% endif %}, via @{{ social_accounts.bluesky ?? 'greenpeace' }} {{ (socialLink ~ '&utm_source=bluesky')|url_encode }}"
```

Comment:
Here (`via @{{ social_accounts.bluesky ?? 'greenpeace' }}`) I added 'greenpeace' as a fallback case.

Reply:
I don't think there is a `greenpeace` account in Bluesky at the moment. Maybe it's better to not have a fallback at all if a Bluesky account is not defined, same as we do for X.

Reply:
Fixed.